Common/src/CConfig.cpp

```diff
@@ -3726,6 +3726,34 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     }
   }
 
+  /*--- Set default values for various fluid properties. ---*/
+
+  const su2double Molecular_Weight_Default = 28.96;
+
+  if (Molecular_Weight == nullptr) {
+    Molecular_Weight = new su2double[1];
+    Molecular_Weight[0] = Molecular_Weight_Default;
+    nMolecular_Weight = 1;
+  }
+
+  /*--- Check whether inputs for FLUID_MIXTURE are correctly specified. ---*/
+
+  if (Kind_FluidModel == FLUID_MIXTURE) {
+
+  /*--- Check whether the number of entries of each specified fluid property equals the number of transported scalar equations solved + 1.
+   nMolecular_Weight is used because it is required for the fluid mixing models. --- */
+    if (nMolecular_Weight != nSpecies_Init + 1) {
+    SU2_MPI::Error("The use of FLUID_MIXTURE requires the number of entries for MOLECULAR_WEIGHT\n"
+                   "to be equal to the number of entries of SCALAR_INIT + 1", CURRENT_FUNCTION);
+    }
+  /*--- Check whether the density model used is correct, in the case of FLUID_MIXTURE the density model must be VARIABLE. Otherwise, if 
+   the density model is CONSTANT, the scalars will not have influence the mixture density and it will remain constant through the complete
+   domain. --- */
+    if (Kind_DensityModel != INC_DENSITYMODEL::VARIABLE) {
+    SU2_MPI::Error("The use of FLUID_MIXTURE requires the INC_DENSITY_MODEL option to be VARIABLE", CURRENT_FUNCTION);
+    }
+  }
```

Comment:
Please format this correctly.

Reply:
Also can this move to the fluid model itself?  

I know NEMO checks for consistency in the fluid model, not sure whats more appropriate.

Reply:
The advantage of doing it in CConfig is that the error will be visible in dry run mode, not sure if the fluid model is instantiated in dry run mode.

Reply:
Done, Thank you so much for your feedback